video.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import datetime
+
+class Video:
```

Comment:
This functionality of this class is very similar to Customer.  Creating a parent class with the base functionality that's similar and then creating child classes with the specifics for the videos and customers can be one way to DRY the code and also simplify writing the code for each child class.